src/lib/node/message_handler.ts

```diff
@@ -31,6 +31,9 @@ export class NodeMessageHandler {
       case NodeCommand.abortFirmwareUpdate:
         await node.abortFirmwareUpdate();
         return {};
+      case NodeCommand.pollValue:
+        const result = await node.pollValue<any>(message.valueId);
```

Comment:
So it looks like this returns the updated value. So this should be called `value` and not `result` ?

Reply:
It *may* return the value if the node responds and polling is supported in the CC.
But yeah, I could rename it to value